.github/actions/dd-ci-upload/action.yml

```diff
@@ -9,9 +9,9 @@ inputs:
     required: false
     default: dd-trace-go
     description: Service name to use with the uploaded test results.
-  files:
+  path:
     required: true
-    description: JUnit files to upload
+    description: Path containing the JUnit files to upload
```

Comment:
Changes to how contribs are tested means that the `ci-test-contrib.sh` script produces more than one JUnitXML file. In order to properly report all of them to the backend, we update this workflow to take in an entire path to a directory (which `junit` will expand and find all XML files within) instead of a single file.